integrations/pinecone/tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 from haystack_integrations.document_stores.pinecone import PineconeDocumentStore
 
 # This is the approximate time it takes for the documents to be available
-SLEEP_TIME = 30
+SLEEP_TIME = 10
```

Comment:
Serverless index (available in the free tier) is faster than pod indexes used previously.